test-framework/kubernetes-client/pom.xml

```diff
@@ -21,10 +21,12 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-server-mock</artifactId>
             <exclusions>
+            <!--
                 <exclusion>
                     <groupId>com.squareup.okhttp3</groupId>
                     <artifactId>okhttp</artifactId>
                 </exclusion>
+                -->
```

Comment:
Either we drop it the exclusion entirely or at least we add a comment explaining the why. Because this is puzzling.